src/sonic-config-engine/minigraph.py

```diff
@@ -555,6 +555,12 @@ def parse_dpg(dpg, hname):
                 vdhcpserver_list = vintfdhcpservers.split(';')
                 vlan_attributes['dhcp_servers'] = vdhcpserver_list
 
+            vintf_node = vintf.find(str(QName(ns, "Dhcpv6Relays")))
+            if vintf_node is not None and vintf_node.text is not None:
```

Comment:
I'd still put code chunk into #8059 . It is better to have each PR to minimal independent functional code changes.

Reply:
OK, will re open the other PR, but this PR can't pass all checks if the other is nor merged.